index.js

```diff
@@ -1,12 +1,14 @@
 'use strict'
 
+require('rootpath')()
```

Comment:
Yes for example

```js
// lib/routes/user.js
var UserAuth = require('auth/user');
var UserModel = require('models/user');
```

Is the model in /, /lib, /server
Is the auth in /, /lib, /server?
etc.. really it get's more confusing then helpful, because it's magic in the background and you have to guess.

I'm still against it. It might be nice for small personal projects, but for something like this I don't think it has a place.